apps/daimo-mobile/src/view/screen/SettingsScreen.tsx

```diff
@@ -325,15 +324,7 @@ function PendingDeviceRow({ slot }: { slot: number }) {
 }
 
 function DetailsSection({ account }: { account: Account }) {
-  const enableNotifications = async () => {
-    await Notifications.requestPermissionsAsync();
-    try {
-      getPushNotificationManager().savePushTokenForAccount();
```

Comment:
@dcposch could you double check these notifs changes don't cause any edge case regression? I was a bit confused why this one didn't go through the extra logic for android in `maybeSavePushTokenForAccount`

Reply:
not sure either. i'll test on android but change lgtm